src/functions/durableEntity.ts

```diff
@@ -0,0 +1,35 @@
+import { HttpRequest, InvocationContext } from '@azure/functions';
+import * as df from 'durable-functions';
+import { DurableOrchestrationClient, IEntityFunctionContext } from 'durable-functions/lib/src/classes';
+
+df.httpClient(
+    'DurableFunctionsEntityHTTPStart',
+    async (_context: InvocationContext, req: HttpRequest, client: DurableOrchestrationClient) => {
```

Comment:
For TypeScript templates I prefer to have the handler defined as a separate function so that a user can reference it in unit tests more easily. See `test/functions/` for an example.

The only caveat is JavaScript, where I prefer to do the handler inline because its the only way you get intellisense and I think that's more important than future tests the user may or may not write.

Reply:
Done